.golangci.yml

```diff
@@ -1,22 +1,23 @@
 linters:
+  disable-all: true
   enable:
     - asciicheck
     - bodyclose
-    - depguard
+    # - depguard
```

Comment:
Rather than disabling these here, so far, we've been adding directives to disable the check on the code, on a case-by-case basis

Reply:
This specific depguard linter requires a "whitelist" of allowed dependencies for use in the binary and testing, so there's no way to disable individual lints in the code.

Reply:
fixed by https://github.com/cometbft/cometbft/pull/1833/commits/fd7dcb442afe35eb7ed1fecaa962c40ac33c67eb